deploy-review/solutions/microservice-session/src/test/java/com/redhat/training/conference/session/SessionResourceTest.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.training.conference.session;
+
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.mockito.InjectMock;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import javax.inject.Inject;
+import com.redhat.training.conference.speaker.SpeakerService;
+
+@QuarkusTest
+public class SessionResourceTest {
```

Comment:
We only have 3 endpoints in the resource, so we might want to have at least 1 test covering each one of them.